fr/cheatsheets/index.md

```diff
@@ -55,12 +55,12 @@ language: fr
 |  `while (x < 5) { println(x); x += 1}`                                                                   |  boucle while. |
 |  `do { println(x); x += 1} while (x < 5)`                                                                |  boucle do while. |
 |  `import scala.util.control.Breaks._`<br>`breakable {`<br>`    for (x <- xs) {`<br>`        if (Math.random < 0.1) break`<br>`    }`<br>`}`|  break. ([transparents](http://www.slideshare.net/Odersky/fosdem-2009-1013261/21)) |
-|  `for (x <- xs if x%2 == 0) yield x*10` _est équivalent à_ <br>`xs.filter(_%2 == 0).map(_*10)`                    |  pour la compréhension : filter/map |
-|  `for ((x,y) <- xs zip ys) yield x*y` _est équivalent à_ <br>`(xs zip ys) map { case (x,y) => x*y }`              |  pour la compréhension : liaison déstructurée |
-|  `for (x <- xs; y <- ys) yield x*y` _est équivalent à_ <br>`xs flatMap {x => ys map {y => x*y}}`                  |  pour la compréhension : produit cartésien. |
-|  `for (x <- xs; y <- ys) {`<br>    `println("%d/%d = %.1f".format(x,y, x*y))`<br>`}`                     |  pour la compréhension : à la manière impérative <br>[sprintf-style](http://java.sun.com/javase/6/docs/api/java/util/Formatter.html#syntax) |
-|  `for (i <- 1 to 5) {`<br>    `println(i)`<br>`}`                                                        |  pour la compréhension : itère jusqu'à la borne supérieure comprise. |
-|  `for (i <- 1 until 5) {`<br>    `println(i)`<br>`}`                                                     |  pour la compréhension : itère jusqu'à la borne supérieure non comprise. |
+|  `for (x <- xs if x%2 == 0) yield x*10` _est équivalent à_ <br>`xs.filter(_%2 == 0).map(_*10)`                    |  compréhension avec for: filter/map |
+|  `for ((x,y) <- xs zip ys) yield x*y` _est équivalent à_ <br>`(xs zip ys) map { case (x,y) => x*y }`              |  compréhension avec for : liaison déstructurée |
+|  `for (x <- xs; y <- ys) yield x*y` _est équivalent à_ <br>`xs flatMap {x => ys map {y => x*y}}`                  |  compréhension avec for : produit cartésien. |
+|  `for (x <- xs; y <- ys) {`<br>    `println("%d/%d = %.1f".format(x,y, x*y))`<br>`}`                     |  compréhension avec for : à la manière impérative <br>[sprintf-style](http://java.sun.com/javase/6/docs/api/java/util/Formatter.html#syntax) |
+|  `for (i <- 1 to 5) {`<br>    `println(i)`<br>`}`                                                        |  compréhension avec for : itère jusqu'à la borne supérieure comprise. |
+|  `for (i <- 1 until 5) {`<br>    `println(i)`<br>`}`                                                     |  compréhension avec for : itère jusqu'à la borne supérieure non comprise. |
```

Comment:
I wonder whether "for en compréhension" would be a more exact translation of "for comprehension". After all, this what we use in math: we can define "un ensemble en compréhension" (the syntax { x € Z | x > -10 }).

But that's French pedantry. TBH when I speak French I say "for comprehension" in English anyway. I would stick to the English term for this.